catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/TransactionRequestConverter.java

```diff
@@ -331,6 +383,15 @@ private Serializable getRecordPropertyValue(HierarchicalStreamReader reader, Str
         return root.getValue();
     }
 
+    public static synchronized JAXBContext getJaxBContext() throws JAXBException {
+        if (jaxBContext == null) {
+
+            jaxBContext = JAXBContext.newInstance("net.opengis.cat.csw.v_2_0_2:"
```

Comment:
You should create a context once.  Actually you should use the Parser API like we do elsewhere in this PR.